crates/aptos-push-metrics/src/lib.rs

```diff
@@ -86,20 +116,71 @@ impl MetricsPusher {
             "Starting push metrics loop. Sending metrics to {} with a frequency of {} seconds",
             push_metrics_endpoint, push_metrics_frequency_secs
         );
+
+        info!(
+            "Execution dashboard link: {} ",
+            Self::get_dashboard_link(chain_name, namespace)
+        );
         Some(thread::spawn(move || {
             Self::worker(
                 quit_receiver,
                 push_metrics_endpoint,
                 push_metrics_frequency_secs,
                 push_metrics_api_token,
+                push_metrics_extra_labels,
             )
         }))
     }
 
+    fn get_dashboard_link(chain_name: &str, namespace: &str) -> Url {
+        let end_time = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis()
+            .to_string();
+        let start_time = SystemTime::now()
+            .sub(Duration::from_secs(600))
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis()
+            .to_string();
+        let mut url =
+            Url::parse("https://o11y.aptosdev.com/grafana/d/execution/execution").unwrap();
```

Comment:
We could get the base URL from env variable as well and use `o11y.aptosdev.com` as default if env isn't available.

Reply:
Addressed this in a separate PR - https://github.com/aptos-labs/aptos-core/pull/6929